lib/electric_slide/agent.rb

```diff
@@ -15,6 +15,15 @@ def initialize(opts = {})
       @presence = opts[:presence] || :available
     end
 
+    def update(attrs = {})
+      attrs.each do |attr, value|
+        setter = "#{attr}="
+        send setter, value if respond_to?(setter)
```

Comment:
Is this intentionally ignoring attributes that don't have setters? This raises the possibility of silent failure